src/modules/setavatar.js

```diff
@@ -0,0 +1,36 @@
+const Eris = require("eris");
+const threadUtils = require("../threadUtils");
+const axios = require("axios");
+
+const validate_img = /(http(s?):)([/|.|\w|\s|-])*\.(?:jpg|gif|png)/;
```

Comment:
Name should be in UPPER_SNAKE_CASE, also the regex provided allows for invalid URLs to be used (ex. `https://asdasd/asdas.png`). I'd recommend using `/^http(s)?:\/\/[\w.-]+(?:\.[\w.-]+)+[\w\-._~:/?#[\]@!$&'()*+,;=.]+\.(png|jpg|gif|webp)$/` instead